quickstart/pom.xml

```diff
@@ -154,9 +154,9 @@
             <artifactId>opentelemetry-exporter-otlp</artifactId>
         </dependency>
         <dependency>
-            <groupId>io.opentelemetry</groupId>
+            <groupId>io.opentelemetry.semconv</groupId>
             <artifactId>opentelemetry-semconv</artifactId>
-            <version>1.30.1-alpha</version>
+            <version>1.23.1-alpha</version>
```

Comment:
Why?

Reply:
https://javadoc.io/static/io.opentelemetry/opentelemetry-semconv/1.30.1-alpha/deprecated-list.html

Reply:
Btw, this is not a downgrade:
https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-semconv
https://mvnrepository.com/artifact/io.opentelemetry.semconv/opentelemetry-semconv